analyzer/codechecker_analyzer/analysis_manager.py

```diff
@@ -57,8 +60,40 @@ def print_analyzer_statistic_summary(metadata_analyzers, status, msg=None):
             LOG.info("  %s: %s", analyzer_type, res)
 
 
-def worker_result_handler(results, metadata_tool, output_path):
-    """ Print the analysis summary. """
+def merge_plists(results, output_path, plist_file_name):
+    """
+    Merge the plist files generated by the analyzers into a single plist file.
+    Deletes the original plist files after merging.
+    """
```

Comment:
Any reason why we are not reusing the method introduced in #4152? That also merged multiple plist files, while also uniquing bug report with the same hash (as generated by `get_report_path_hash()`, not by the analyzer).